### ISSUE TITLE
Add "button" accessibility role to all buttons

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#558]
Location of missing button labels:
- Take control of your information (“Get Started”)
- EULA (“Continue” and “Back”)
- Create PIN (“Create PIN”)
- New Credential Offer (“View”)
- New Credential Offer (“Accept” and “Decline”)
- Credential added to your wallet (“Done”)
- Please enter your PIN (“Enter”)
- Incorrect PIN (“Okay”)

**Comment by bryce-mcmath** [comment]
@knguyenBC Do we want touchable icons to have the button role as well? The "Home" and "Back" and "Add credential" icon buttons for example? Do we want touchable text (ie. "Get BC Services Card App" text) to have the button role as well?

**Comment by knguyenBC** [comment]
The button role should be used for elements that trigger a response when activated by the user. So touchable icons should have a button role as well. That include "Home", "Back", "Add credential", "Hide/Show", "Info", "More", "Close/Dismiss" etc.

**Comment by knguyenBC** [comment]
The button role should be used for elements that trigger a response when activated by the user. So touchable icons should have a button role as well. That include "Home", "Back", "Add credential", "Hide/Show", "Info", "More", "Close/Dismiss" etc.
